### PR TITLE
bump ES plugin to 2.4.4.12 and add docs regarding access control

### PR DESCRIPTION
**Pull request by jcantrill**
This PR:
* bumps plugin to 2.4.4.12 to pull in user based roles
* adds documentation regarding how the openshift plugin creates roles and role mappings.

### CODE REVIEW

**Comment by jcantrill** [comment]
[test]

**Comment by jcantrill** [comment]
I actually thought we had this in 3.6.  We may need to backport this to satisfy atleast one BZ (I think) that was logged against the inability to cross-query namespaces. @richm ? @portante ?

File: docs/access-control.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      '*': [INDEX_ANY_OPERATIONS]
+    ?operations?:
+      '*': [INDEX_OPERATIONS]
+gen_user_user2_bar_email_com:
+  indices:
+    .all_user2_bar_email_com:
+      '*': [INDEX_PROJECT]
+    ?kibana?994a33f6a157ba4a286395f81a4333db1e6cefb6:
+      '*': [INDEX_KIBANA]
+    ?project?foo?bar?*:
```
Comment (richm) [inline docs/access-control.md]: these don't need to begin with `?`

File: docs/access-control.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      '*': [INDEX_PROJECT]
+    ?project?xyz?*:
+      '*': [INDEX_PROJECT]
+    foo?bar?*:
```
Comment (richm) [inline docs/access-control.md]: should indicate that these are for legacy purposes only

File: elasticsearch/Dockerfile.centos7 [hunk 3]
```diff
@@ -43,7 +43,8 @@ RUN rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch && \
                 PyYAML && \
     yum clean all
 
-
+ENV OSE_ES_URL=file:///openshift-elasticsearch-plugin-2.4.4.13-SNAPSHOT.zip
+ADD openshift-elasticsearch-plugin-2.4.4.13-SNAPSHOT.zip /
```
Comment (richm) [inline elasticsearch/Dockerfile.centos7]: I'm assuming this is only for testing purposes and will be removed from the final PR before merging

Comment (jcantrill) [reply elasticsearch/Dockerfile.centos7]: grr...

File: docs/access-control.md (hunk 1 above)
Comment (jcantrill) [reply docs/access-control.md]: This is a straight port from refactoring of the roles.  I believe the ? also means the dot is optional and I wondered why we had it too.  We will need to make a change to the plugin to correct.  I'll update the documentation to remove it for now

**Comment by richm** [comment]
[merge]

**Pull request merged**